app/controllers/bill_controller.rb

```diff
@@ -0,0 +1,26 @@
+class BillController < ApplicationController
+
+  def index
+    @bills = Bill.all
+  end
+
+  def new
+    @bill = Bill.new
+  end
+
+  def create
```

Comment:
line 11: Not sure how we want to fill this out. Will this be a request from the merchant? We could probably just create a bill and redirect to the show page? redirect_to @bill? I don't know. This could also be where we send the bill as json to the phone at this point. Something to discuss.